.github/workflows/cypress_workflow.yml

```diff
@@ -146,70 +203,93 @@ jobs:
       - run: npx cypress cache list
       - run: npx cypress cache path
 
-      # Setup spec files for Dashboards in-house cypress tests
-      - name: Setup spec files for Dashboards tests
-        if: ${{ inputs.specs == '' && matrix.group > 9 }}
+      # Run tests based on configuration
+      - name: Run FT repo tests with no query enhancements
+        if: matrix.test_location == 'ftr' && matrix.config == 'standard'
+        uses: cypress-io/github-action@v2
+        with:
+          working-directory: ${{ env.FTR_PATH }}
+          start: ${{ env.OPENSEARCH_SNAPSHOT_CMD }}, ${{ env.START_CMD }}
+          wait-on: 'http://localhost:9200, http://localhost:5601'
+          command: yarn cypress:run-without-security --browser ${{ env.CYPRESS_BROWSER }} --spec ${{ env.SPEC }}
+
+      - name: Download OpenSearch
+        if: matrix.test_location == 'ftr' && matrix.config == 'query_enhanced'
+        uses: suisei-cn/actions-download-file@v1.4.0
+        with:
+          url: https://artifacts.opensearch.org/releases/bundle/opensearch/${{ env.LATEST_VERSION }}/opensearch-${{ env.LATEST_VERSION }}-linux-x64.tar.gz
+
+      - name: Extract OpenSearch
+        if: matrix.test_location == 'ftr' && matrix.config == 'query_enhanced'
+        run: |
+          tar -xzf opensearch-*.tar.gz
+          rm -f opensearch-*.tar.gz
         shell: bash
+
+      - name: Remove security plugin
+        if: matrix.test_location == 'ftr' && matrix.config == 'query_enhanced'
         run: |
-          IFS="," read -a SPEC_ARRAY <<< $(yarn --silent osd:ciGroup${{ matrix.group }})
-          DASHBOARDS_SPEC=''
-          for i in "${SPEC_ARRAY[@]}"; do
-          DASHBOARDS_SPEC+="cypress/integration/core_opensearch_dashboards/${i},"
-          done
-          echo "DASHBOARDS_SPEC=${DASHBOARDS_SPEC}" >> $GITHUB_ENV
+          /bin/bash -c "yes | ./opensearch-${{ env.LATEST_VERSION }}/bin/opensearch-plugin remove opensearch-security"
+        shell: bash
 
-      # Run FT repo tests
-      - name: Run FT repo tests
-        if: ${{ matrix.group < 10 }}
+      - name: Run OpenSearch
+        if: matrix.test_location == 'ftr' && matrix.config == 'query_enhanced'
+        run: |
+          /bin/bash -c "./opensearch-2.17.0/opensearch-tar-install.sh &"
+          sleep 30
```

Comment:
confused about this one.

wait-on should be actually a better resolution to this: https://github.com/cypress-io/github-action?tab=readme-ov-file#wait-on

are you seeing issues with that?